frontend/src/app/general/header/header.component.ts

```diff
@@ -15,11 +16,15 @@ import { UserService } from '../../services/user/user.service';
 export class HeaderComponent implements OnInit {
   constructor(
     public authService: AuthService,
-    public userService: UserService
+    public userService: UserService,
+    public navBarService: NavBarService
   ) {}
 
+  isSmallScreen = false;
```

Comment:
Instead of using this, I would use `@media` queries in CSS.

Reply:
Good point. Never used `@media` queries before, but they look really simple. 

I'd wait until we decide if we want to add Tailwind CSS (https://github.com/DSD-DBS/capella-collab-manager/pull/767). Tailwind CSS provides easy facilities to wrap the media queries.

Reply:
Merged tailwind and replace it with simple `xl:hidden`.